packages/gatsby-theme-newrelic/src/icons/feather.js

```diff
@@ -21,6 +21,11 @@ export default {
       <line x1="10" y1="14" x2="21" y2="3" />
     </>
   ),
+  filter: (
+    <>
+      <polygon points="22 3 2 3 10 12.46 10 19 14 21 14 12.46 22 3" />
+    </>
```

Comment:
```suggestion
    <polygon points="22 3 2 3 10 12.46 10 19 14 21 14 12.46 22 3" />
```

Since this one only has a single child, no need for the fragment around it 😄